Cesium.Compiler/Main.cs

```diff
@@ -30,7 +30,9 @@
 
         var cesiumRuntime = args.CesiumCRuntime ?? Path.Combine(AppContext.BaseDirectory, "Cesium.Runtime.dll");
         var defaultImportsAssembly = args.DefaultImportAssemblies ?? Array.Empty<string>();
+#pragma warning disable IL3000
```

Comment:
👍

Reply:
Please put an explanation in a comment here. Otherwise, it's hard to review and/or refactor if the code moves.

Reply:
```suggestion
#pragma warning disable IL3000 Automatic discovery of corelib is fallback option, if tooling do not pass that parameter
```